tools/Az.Tools.Predictor/Az.Tools.Predictor/command_param_to_resource_map.json

```diff
@@ -1 +1 @@
"iothubdistributedtracing": {"deviceid": "deviceid", "iothubname": "iothubname", "resourcegroupname": "resourcegroupname", "resourceid": "resourceid"}, "iothubmoduleconnectionstring": {"deviceid": "deviceid", "iothubname": "iothubname", "moduleid": "moduleid", "resourcegroupname": "resourcegroupname", "resourceid": "resourceid"}, "iothubmoduletwin": {"deviceid": "deviceid", "iothubname": "iothubname", "moduleid": "moduleid", "resourcegroupname": "resourcegroupname", "resourceid": "resourceid"}, "iothubdevicemethod": {"deviceid": "deviceid", "iothubname": "iothubname", "name": "iothubdevicemethodname", "payload": "payload", "resourcegroupname": "resourcegroupname", "resourceid": "resourceid"}, "applyupdate": {"applyupdatename": "applyupdatename", "providername": "providername", "resourcegroupname": "resourcegroupname", "resourcename": "resourcename", "resourceparentname": "resourceparentname", "resourceparenttype": "resourceparenttype", "resourcetype": "resourcetype"}, "configurationassignment": {"providername": "providername", "resourcegroupname": "resourcegroupname", "resourcename": "resourcename", "resourceparentname": "resourceparentname", "resourceparenttype": "resourceparenttype", "resourcetype": "resourcetype", "configurationassignmentname": "configurationassignmentname", "location": "location", "maintenanceconfigurationid": "maintenanceconfigurationid", "resourceid": "resourceid"}, "maintenanceconfiguration": {"name": "maintenanceconfigurationname", "resourcegroupname": "resourcegroupname", "location": "location", "maintenancescope": "maintenancescope", "expirationdatetime": "expirationdatetime", "recurevery": "recurevery", "startdatetime": "startdatetime", "timezone": "timezone", "visibility": "visibility"}, "maintenanceupdate": {"providername": "providername", "resourcegroupname": "resourcegroupname", "resourcename": "resourcename", "resourceparentname": "resourceparentname", "resourceparenttype": "resourceparenttype", "resourcetype": "resourcetype"}, "insightsprivatelinkscope": {"name": "insightsprivatelinkscopename", "resourcegroupname": "resourcegroupname", "resourceid": "resourceid", "location": "location"}, "insightsprivatelinkscopedresource": {"name": "insightsprivatelinkscopedresourcename", "resourcegroupname": "resourcegroupname", "resourceid": "resourceid", "scopename": "scopename", "linkedresourceid": "linkedresourceid"}, "p2svpngatewayvpnconnection": {"resourceid": "resourceid"}, "virtualnetworkgatewayvpnconnection": {"resourcegroupname": "resourcegroupname", "resourcename": "virtualnetworkgatewayname", "virtualnetworkgatewayname": "virtualnetworkgatewayname", "resourceid": "resourceid"}, "privatelinkresource": {"privatelinkresourceid": "privatelinkresourceid", "privatelinkresourcetype": "privatelinkresourcetype", "resourcegroupname": "resourcegroupname", "servicename": "servicename"}, "applicationgatewayfirewallpolicyexclusion": {"matchvariable": "matchvariable", "selector": "selector", "selectormatchoperator": "selectormatch
```

Comment:
Not sure if I'm supposed to update this manually, or if there's a way to auto-generate this.